src/Engine/ProtoCore/Properties/Resources.resx

```diff
@@ -259,37 +259,37 @@
     <value>Multiple definitions for '{0}' are found as {1}</value>
   </data>
   <data name="kNoStepSizeInAmountRangeExpression" xml:space="preserve">
-    <value>No step size is specified in amount range expression.</value>
+    <value>No step size specified in amount-based range expression.</value>
   </data>
   <data name="kPropertyInaccessible" xml:space="preserve">
-    <value>Property '{0}' is inaccessible.</value>
+    <value>Property '{0}' is inaccessible from here.</value>
   </data>
   <data name="kPropertyIsInaccessible" xml:space="preserve">
-    <value>Property '{0}' is inaccessible.</value>
+    <value>Property '{0}' is inaccessible from here.</value>
   </data>
   <data name="kPropertyNotFound" xml:space="preserve">
-    <value>Property '{0}' not found.</value>
+    <value>Property '{0}' was not found.</value>
   </data>
   <data name="kPropertyOfClassNotFound" xml:space="preserve">
     <value>Class '{0}' does not have a property '{1}'.</value>
   </data>
   <data name="kRangeExpressionConflictOperator" xml:space="preserve">
-    <value>The amount operator cannot be used together with step operator.</value>
+    <value>An amount operator cannot be used together a step operator.</value>
   </data>
   <data name="kRangeExpressionWithInvalidAmount" xml:space="preserve">
-    <value>The amount of step is invalid.</value>
+    <value>The amount of the step was invalid.</value>
   </data>
   <data name="kRangeExpressionWithInvalidStepSize" xml:space="preserve">
-    <value>The step size of range expression is invalid.</value>
+    <value>The step size was invalid.</value>
   </data>
   <data name="kRangeExpressionWithNegativeStepNumber" xml:space="preserve">
-    <value>The step number of range expression should be greater than 0.</value>
+    <value>The step number must be greater than 0.</value>
   </data>
   <data name="kRangeExpressionWithNonIntegerStepNumber" xml:space="preserve">
-    <value>The step number of range expression should be integer.</value>
+    <value>The step number must be an integer.</value>
   </data>
   <data name="kRangeExpressionWithStepSizeZero" xml:space="preserve">
-    <value>The step size of range expression should not be 0.</value>
+    <value>The step size of must not be 0.</value>
```

Comment:
Well, it's kinda lonely out here, so I've taken to talking to myself on code reviews.

But yeah - it's much easier to reread the changes here, catch a few issues, bundle them all up into a single patch than to do the whole change, push palaver to remove an 'of '.